pygpu/gpuarray.pxd

```diff
@@ -172,10 +172,6 @@ cdef extern from "gpuarray/array.h":
                           gpudata *data, size_t offset, int typecode,
                           unsigned int nd, const size_t *dims,
                           const ssize_t *strides, int writable)
-    int GpuArray_copy_from_host(_GpuArray *a,
```

Comment:
There was some bad values, but it's also an ill-concieved idea from the start.

The idea was to avoid copying data on the CPU during a transfer so it would figure out the start and the end of the passed-in data (using strides and shapes) and copy the whole block.  This would potentially copy large swaths of irrelevant data that just happen to sit between those boundaries.

It also bypassed the forced offset that is added in DEBUG and create some problems related to that.